dev/com.ibm.websphere.appserver.features/visibility/public/mcp-1.0/io.openliberty.mcp-1.0.feature

```diff
@@ -0,0 +1,23 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mcp-1.0
+visibility=public
+IBM-App-ForceRestart: install, \
+ uninstall
+IBM-API-Package: \
+ io.openliberty.mcp, \
+ io.openliberty.mcp.annotations, \
+ io.openliberty.mcp.content, \
+ io.openliberty.mcp.encoders, \
+ io.openliberty.mcp.messaging, \
```

Comment:
```suggestion
 io.openliberty.mcp.annotations, \
 io.openliberty.mcp.content, \
 io.openliberty.mcp.encoders, \
```
Those two packages don't seem to exist.

Reply:
Messaging will be added in #32521, I'll remove the other one.